File: distros/J/JavonetPerlSdk/lib/Javonet/Binaries/Nodejs/core/generator/handler/shared/SharedArgumentNameHandler.js
const SharedHandlerType = require("../../internal/SharedHandlerType");
const AbstractGeneratorHandler = require("../AbstractGeneratorHandler");

class SharedArgumentNameHandler extends AbstractGeneratorHandler {

    generate_code(existing_string_builder, common_command, used_object, handlers) {
        handlers.SHARED_HANDLER[SharedHandlerType.METHOD_NAME].generate_code(existing_string_builder, common_command, used_object, handlers)
    }
}

module.exports = SharedArgumentNameHandler

File: distros/J/JavonetPerlSdk/lib/Javonet/Binaries/Python/javonet/core/generator/handler/shared/SharedArgumentNameHandler.py
from javonet.core.generator.handler.AbstractGeneretatorHandler import AbstractGeneratorHandler
from javonet.core.generator.internal.SharedHandlerType import SharedHandlerType


class SharedArgumentNameHandler(AbstractGeneratorHandler):
    def generate_command(self, analyzed_object, parent_command, handlers):
        handlers.SHARED_HANDLER[SharedHandlerType.METHOD_NAME].generate_command(analyzed_object, parent_command,
                                                                                handlers)

    def generate_code(self, existing_string_builder, common_command, used_object, handlers):
        handlers.SHARED_HANDLER[SharedHandlerType.METHOD_NAME].generate_code(existing_string_builder, common_command,
                                                                             used_object, handlers)
